src/sass/_normalize.scss

```diff
@@ -0,0 +1,12 @@
+// Ideally all of this will go in uswds/src/stylesheets/lib/normalize, but until
+//  it does, normalizing css goes in here.
+
+dl {
```

Comment:
I think this is ok, but you should check the other uses of `<dl>` to make sure it looks ok (looks like there are only a few).